src/core/geometry/qgsgeometryutils.h

```diff
@@ -93,6 +93,11 @@ class CORE_EXPORT QgsGeometryUtils
      */
     static double sqrDistToLine( double ptX, double ptY, double x1, double y1, double x2, double y2, double &minDistX SIP_OUT, double &minDistY SIP_OUT, double epsilon ) SIP_HOLDGIL;
 
+    /**
+     * Returns the distance between a point and an infinite line.
+     */
+    static double distToInfiniteLine( double ptX, double ptY, double x1, double y1, double x2, double y2, double epsilon = 1e-7 );
```

Comment:
wouldn't it be a nicer API to have `distToInfiniteLine( const QgsPointXY &point, const QgsPointXY &linePoint1, const QgsPointXY &linePoint2, double epsilon = 1e-7)` at the cost of consistency?